docs/src/main/sphinx/language/reserved.md

```diff
@@ -89,3 +92,32 @@ be quoted (using double quotes) in order to be used as an identifier.
 | `WHERE`             | reserved | reserved |
 | `WITH`              | reserved | reserved |
 
+(language-identifiers)=
+## Identifiers
+
+Tokens that identify names of catalogs, schemas, tables, columns, functions, or
+other objects, are identifiers.
+
+Identifiers must start with a letter, and subsequently include alphanumeric
+characters and underscores. Identifiers with other characters must be delimited
+with double quotes (`"`). When delimited with double quotes, identifiers can use
```

Comment:
This should mention how to escape a double quote in a quoted identifier.

Reply:
its `\"` right?

Reply:
No, you escape a `"` with another double quote. I.e., `""`. This analogous to how single quotes are escaped in string literals.

Reply:
It's double the double quote